projects/storefrontlib/src/cms-components/product/product-tabs/product-reviews/product-reviews.component.ts

```diff
@@ -18,11 +24,14 @@ export class ProductReviewsComponent {
   maxListItems: number;
   reviewForm: FormGroup;
 
-  product$: Observable<Product> = this.currentProductService.getProduct();
+  product$: Observable<Product> = this.currentProductService
+    .getProduct()
+    .pipe(filter(p => !!p));
```

Comment:
let's keep it in `reviews$` as it could be considered breaking (current behavor of public property)